contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2448,6 +2448,13 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 								if (windows_login_contains_invalid_chars(orig_loginname))
 									ereport(ERROR, (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
 													errmsg("'%s' is not a valid name because it contains invalid characters.", orig_loginname)));
+								
+								/*
+								 * Check whether the domain name contains invalid characters or not.
+								 */
+								if (windows_domain_contains_invalid_chars(orig_loginname))
+									ereport(ERROR, (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+													errmsg("'%s' is not a valid domain because it contains invalid characters.", orig_loginname)));
```

Comment:
better to use login?

Reply:
From discussion with DBE:
User query: CREATE LOGIN [domiain@%\loginname] FROM WINDOWS
Err Msg: "domiain@%\loginname is not valid because the domain name contains invalid characters"